src/styles.js

```diff
@@ -123,7 +127,8 @@ export default css`
     opacity: 100%;
   }
 
-  .vacuum.docked {
+  .vacuum.docked,
+  .vacuum.idle {
```

Comment:
This was intentional. Idle without opacity has to indicate that vacuum is not in the proper place.

Reply:
Gotcha. I will fix this.

Reply:
Current push should resolve this.